evil-nerd-commenter.el

```diff
@@ -326,7 +326,9 @@ See http://lists.gnu.org/archive/html/bug-gnu-emacs/2013-03/msg00891.html."
 
        ;; Emacs 26.1
        ((fboundp 'org-element-at-point)
-        (setq lang (org-element-property :language (org-element-at-point))))))
+        (setq lang
+              (let ((lang (org-element-property :language (org-element-at-point))))
```

Comment:
Merge two lines into one line, 
```lisp
(or (cdr (assoc lang org-src-lang-modes))
    (org-element-property :language (org-element-at-point)))
```
Besides, please add unit test in `evil-nerd-commenter-tests.el` to test `evilnc--org-lang-major-mode`.

Reply:
I do not think this works as we first need to get lang from the name to look it up in `org-src-lang-modes` and additionally we need it as the fallback incase it is not in `org-src-lang-modes`.
So I think it makes sense to first calling `(org-element-property :language (org-element-at-point))` and setting `lang` to it and then choosing the right one.
Or did I understand something wrong?

The test is added.

Reply:
OK, code style is not major issue here. I accept your request at first.

Reply:
Thank you. 
Just to better my emacs lisp, what is the code style problem?

Reply:
two variable with same name `lang`.

Reply:
Ah ok, this I would have understood better.